CONTRIBUTING.md

```diff
@@ -145,3 +145,60 @@ Release tag are only done on the `main` branch.
     * `release`, used for releasing project
     * `hotfix`, used for applying a hotfix on main
 * `BRANCH_NAME` is managed by this regex: `[a-z0-9._-]` (`_` is used as space character).
+
+### Testing OIDC Authentication
+
+If you want to test an authentication with an OIDC provider, you need to set up leto-modelizer in HTTPS to test the connection with a provider. To do so, you have to change the dev server type, port, add a proxy and change `vueRouterMode` from `hash` to `history` inside `quasar.config.js`. Here is an example with Github provider:
+
+```
+build: {
+  vueRouterMode: 'history', // available values: 'hash', 'history'
+  ...
+},
+
+devServer: {
+  server: {
+    type: 'https',
+  },
+  port: 443,
+  '/auth': {
+    target: 'https://token.actions.githubusercontent.com/',
+    changeOrigin: true,
+    pathRewrite: {
+      '^/auth': '',
+    },
+    secure: true,
+    onProxyRes(proxyRes) {
+      proxyRes.headers['Access-Control-Allow-Origin'] = '*';
+    },
+  },
+  '/token': {
+    target: 'https://github.com/login/oauth/access_token/',
```

Comment:
add something : `// url to the oidc token of your provider`

Reply:
done